transport_plugins/jlink/iotile_transport_jlink/jlink.py

```diff
@@ -318,7 +372,7 @@ def _parse_conn_string(self, conn_string):
         response = await self._jlink_async.send_rpc(
             self._device_info, self._control_info,
             address, rpc_id, payload, timeout)
-        return response['payload']
+        return response['payload'] if response else None
```

Comment:
See comment below, either `bytes` should be returned (possibly 0 length) or an exception should be raised.

Reply:
Fixed it